tests/visualization_tests/matplotlib_tests/test_optimization_history.py

```diff
@@ -88,6 +88,18 @@ def objective(trial: Trial) -> float:
         else:
             assert legend.get_text() == f'Objective Value of {studies[i-n_studies].study_name}'
 
+
+    # Test customized target.
+    with pytest.warns(UserWarning):
+        figure = plot_optimization_history(studies, target=lambda t: t.number)
+    assert len(figure.get_lines()) == 0
+    assert len(figure.get_legend().get_texts()) == n_studies
+
+    # # Test customized target name.
```

Comment:
Oh, this is my bad. Could you remove this?

```suggestion
    # Test customized target name.
```